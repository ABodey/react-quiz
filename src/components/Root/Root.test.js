import React from 'react'
import { shallow } from 'enzyme'
import Root from './Root'

describe('<Root />', () => {
    it ('should match snapshots', () => {
      const wrapper = shallow (<Root />)

      expect(wrapper).toMatchSnapshot()
    })
})
