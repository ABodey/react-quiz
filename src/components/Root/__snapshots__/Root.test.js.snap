// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Root /> should match snapshots 1`] = `<App />`;
